indico/modules/events/timetable/templates/pdf/timetable.css

```diff
@@ -130,19 +130,19 @@ body {
 
 /* Main content */
 .session-block.contribution .inherited-bg-color {
-  background-color: #af4571;
+  background-color: rgb(146, 182, 219);
 }
 
 .session-block.contribution .inherited-bg-color-trans {
-  background-color: #af457166;
+  background-color: rgb(146, 182, 219, 66);
```

Comment:
There's no `#rrggbbaa` hex notation? :/

Reply:
I am not using hex and also changed it to an even more fitting blue as it is the same color as in the table of contents:

![Screenshot from 2025-05-13 11-27-06](https://github.com/user-attachments/assets/fc883f7e-a318-475f-8dbd-0bf9561b0a29)